### ISSUE TITLE
Helper_math.h header file missing from cuda-samples

### ISSUE THREAD

**Issue by rickstaa** [NVIDIA/cuda-samples#15]
# Problem description
I am currently trying to make the (OpenKinect/libfreenect2)[https://github.com/OpenKinect/libfreenect2] inside my `From: nvidia/cuda:10.0-cudnn7-devel-ubuntu16.04` singularity image. While building I however get the following error message.

```
/opt/libfreenect2/src/cuda_kde_depth_packet_processor.cu:39:25: fatal error: helper_math.h: No such file or directory
compilation terminated.
CMake Error at cuda_compile_generated_cuda_kde_depth_packet_processor.cu.o.cmake:207 (message):
  Error generating
  /opt/libfreenect2/build/CMakeFiles/cuda_compile.dir/src/./cuda_compile_generated_cuda_kde_depth_packet_processor.cu.o


CMakeFiles/freenect2.dir/build.make:80: recipe for target 'CMakeFiles/cuda_compile.dir/src/cuda_compile_generated_cuda_kde_depth_packet_processor.cu.o' failed
make[2]: *** [CMakeFiles/cuda_compile.dir/src/cuda_compile_generated_cuda_kde_depth_packet_processor.cu.o] Error 1
CMakeFiles/Makefile2:136: recipe for target 'CMakeFiles/freenect2.dir/all' failed
make[1]: *** [CMakeFiles/freenect2.dir/all] Error 2
Makefile:127: recipe for target 'all' failed
make: *** [all] Error 2
```

I assume this is caused by the fact that the docker does not install the cuda-samples by default. I tried using this repository to solve the error but the `helper_math.h` header file appears not to be present in the repository. I therefore currently use the repository of [zchee](https://github.com/zchee/cuda-sample) to achieve this. Is it maybe possible to add these header files to the repository so I can use the official repository instead?

# Related topics
- [Missing samples](https://github.com/NVIDIA/cuda-samples/issues/9)

**Comment by mdoijade** [comment]
@rickstaa helper_math.h is not been added to this repo because no existing samples in it currently needs it.  
In future when such samples are added we would add it.

**Comment by nathantsoi** [comment]
@mdoijade, just an FYI -- it would be useful to have all the headers here, as other packages depend on these files: e.g. https://github.com/OpenKinect/libfreenect2/issues/777#issuecomment-480023146

and installing from the runfile is inconvenient in a docker container, for example.

**Comment by mdoijade** [comment]
@nathantsoi @rickstaa helper_math.h is now added to the repo. https://github.com/NVIDIA/cuda-samples/tree/10.1.2

**Comment by rickstaa** [comment]
@mdoijade First of all, thanks a lot for including the `helper_math.h`.  I however still had one problem when installing a package which has the headers as a dependency. This is caused by a difference in the sub-folder names of the NVIDIA_CUDA-10.0_Samples folder that is included with the drivers and the one present in this repository. Due to this difference, the packages using these headers still throw an exception. I now solve this by manually renaming the folders, but it might be helpful to change this to improve consistency.

#### Included NVIDIA_CUDA-10.0_Samples tree
```
.
├── 0_Simple
├── 1_Utilities
├── 2_Graphics
├── 3_Imaging
├── 4_Finance
├── 5_Simulations
├── 6_Advanced
├── 7_CUDALibraries
├── common
├── EULA.txt
└── Makefile
```

#### This repository

``
.
├── Common
├── LICENSE
├── Makefile
├── README.md
└── Samples
```
